applications/ziggurat.md

```diff
@@ -0,0 +1,266 @@
+
+For detailed information on this implementation, see the [Ziggurat manual].
+
+[Ziggurat]: https://github.com/eqlabs/ziggurat
+[Ziggurat manual]: (https://fluttering-gem-83c.notion.site/The-Ziggurat-Manual-88f6ab7d85774c5fb1bc12bcd64bf700)
+
+## Development Roadmap :nut_and_bolt:
+
+This project has one main deliverable: the test suite. However, this deliverable is prefaced by a pre-flight check phase of requirement gathering and liasing with node developers.
+
+### 1. Pre-flight Checks
+
+Collaborate with the Polkadot node devs to determine the final requirements, and do any necessary setup / housekeeping on the node
+
+- Starting with Equilibrium's baseline spec, we will work with the node developers to finalize the specification, including test scenarios, potential new RPC endpoints, and the output format.
+- Perform initial repository setup and housekeeping
+
+### 2. Develop Test Suite
+Create a suite of black box tests for conformance to spec, performance bottlenecks, and resistance to malicious actors
+
+- Start at the lowest level of network activity
+- Move through the network protocol piece by piece to ensure conformance to spec
+- Introduce a networking-only test node for "black box" coverage
+- Introduce stress testing and enhanced fuzzing
+- Introduce tests for malicious behavior, such as:
+  - Sending messages with flipped bits or other corruption
+  - Spawning a vast amount of connections or reconnecting many times
+  - Spamming messages
+
+## Budget and Schedule
+
+### Milestone Deliverables
+
+We will split the grant process into two milestones: one for requirement gathering and solidification, and another for the final test suite deliverables.
+
+#### Milestone 1: Requirement Gathering
+
+The Equilibrium team will liase with the Polkadot node developers to determine the best course of action, testing metholody, and testing targets for Ziggurat, culminating in a specification document to be followed in Milestone 2.
+
+- **Total Estimated Duration:** One month
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** $62,000
+
+The grant template deliverables 0a-0d are not required for this milestone as this is entirely communications and research-based. Indeed, you could consider the Zigguarat and license commensurate in this milestone for their corresponding deliverables whereas the Docker image and test suite itself are likewise commensurate in Milestone 2. More detail in the following milestone.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1a. | Ziggurat Spec for Polkadot | This will be in the form of the form of the [ZCash spec] |
+| 1b. | Open Source License | Ziggurat is currently dual-licensed under the exceedingly permissive Apache + MIT combination and will continue to be. |
```

Comment:
```suggestion
| 0a. | License | Ziggurat is currently dual-licensed under the exceedingly permissive Apache + MIT combination and will continue to be. |
| 1. | Ziggurat Spec for Polkadot | This will be in the form of the form of the [ZCash spec] |

```